### DISCUSSION TITLE
Much higher prices and they don't change

### DISCUSSION THREAD

**Discussion by ghost** [JaccoR/hass-entso-e#109] [General]
![Prijzen](https://user-images.githubusercontent.com/15307842/214393772-97a85203-d236-4722-87f7-0f235914a89b.png)

I get this high prices, and they don't chance, even if i change the prizes. i use this:
{% set s = {"extra_cost": 0.02118,"energie_belasting": 0.15246,"ode_belasting": 0.0} %} {{(current_price + s.extra_cost + s.energie_belasting + s.ode_belasting) | float}}

**Comment by cyborghound** [comment]
This is also happening to me, no matter what I change, the prices stay the same.
